### PR TITLE
interface: removed unused links files

### PR DESCRIPTION
**Pull request by tacryt-socryp**

### CODE REVIEW

File: pkg/interface/src/views/apps/links/components/link-item.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const LinkItem = (props) => {
     ? <img src={props.avatar} height={36} width={36} className="dib" />
     : <Sigil ship={`~${author}`} size={36} color={'#' + props.color} />;
 
-  const baseUrl = props.baseUrl || `/~link/${resource}`;
+  const baseUrl = props.baseUrl || `/~404/${resource}`;
```
Comment (matildepark) [inline pkg/interface/src/views/apps/links/components/link-item.js:36]: Why not just `props.baseUrl`

**Review by matildepark — APPROVED** [review]

**Comment by matildepark** [comment]
Hang on for Publish until I PR some fixes for a few crashes shortly

File: pkg/interface/src/views/apps/links/components/link-item.js (hunk 1 above)
Comment (tacryt-socryp) [reply pkg/interface/src/views/apps/links/components/link-item.js:36]: This is in the case that props.baseUrl is null or undefined. Better to explicitly redirect to /~404 imo

**Comment by matildepark** [comment]
Ok ping when you're ready to merge this

**Comment by tacryt-socryp** [comment]
It's ready to go.

**Pull request merged**